addons/point_of_sale/static/src/css/pos_receipts.css

```diff
@@ -21,6 +21,7 @@
 
 .pos-receipt .pos-receipt-left-padding {
     padding-left: 2em;
+    word-break: break-all;
```

Comment:
needed only for notes, or something else?

Reply:
Done. I've made a new class for it